.github/workflows/build-images-base.yaml

```diff
@@ -91,6 +96,7 @@ jobs:
           build-args: |
             GIT_SHA=${{ github.sha }}
             DIRTY=false
+            VERSION=${{ steps.release.outputs.version }}
```

Comment:
Here you can pass the `inputs.operatorVersion`, no?

Reply:
sure!

Reply:
done!